### PR TITLE
Switch this repo to the build-tools container.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -178,15 +179,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
```
Comment (sdake) [inline LICENSE]: this looks like a slightly derived license.  We should take a look at not deriving the license if possible.

File: LICENSE [hunk 2]
```diff
@@ -178,15 +179,15 @@
...
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2016-2019 Istio Authors
```
Comment (sdake) [inline LICENSE]: Note the CNCF legal eagles have posted a nice MD doc describing how copyrights should be handled.  I agree with everything in the document.  I'd recommend taking a look here:

https://github.com/cncf/foundation/blob/master/copyright-notices.md

File: scripts/run_gobindata.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOTDIR=$(dirname "${SCRIPTPATH}")
+
+img=gcr.io/istio-testing/api-build-tools:2019-07-30
```
Comment (sdake) [inline scripts/run_gobindata.sh]: api-build-tools is retired.

File: scripts/run_gobindata.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+docker run -i --sig-proxy=true --rm --entrypoint go-bindata --user "$(id -u)" -v /etc/passwd:/etc/passwd:ro -v "${ROOTDIR}:${ROOTDIR}" -w "$(pwd)" ${img} "$@"
```
Comment (sdake) [inline scripts/run_gobindata.sh]: if this run_gobindata.sh is run by this repo, the repo probably won't work in a container.

Comment (geeknoid) [reply scripts/run_gobindata.sh]: Indeed. The script will be retired once all repos are on BUILD_WITH_CONTAINER=1

File: scripts/run_gobindata.sh (hunk 3 above)
Comment (geeknoid) [reply scripts/run_gobindata.sh]: Point-in-time stuff. I updated the common files again, so now it's pointing to the new image.

File: LICENSE (hunk 1 above)
Comment (geeknoid) [reply LICENSE]: The LICENSE file that is in istio/common-files (which is being copied here) is the byte-for-byte copy of the original text from https://www.apache.org/licenses/LICENSE-2.0.txt, with the exception to the updated copyright date and copyright owner name on line 189.

File: LICENSE (hunk 2 above)
Comment (geeknoid) [reply LICENSE]: See https://github.com/istio/common-files/pull/34 for an update that removes the date range.

File: scripts/run_gobindata.sh (hunk 3 above)
Comment (sdake) [reply scripts/run_gobindata.sh]: cool - although I believe the sscript will need more attention to run properly with the container variable set.  I'll take a look at the common PRs.

File: LICENSE [hunk 5]
```diff
@@ -1,4 +1,5 @@
-                                 Apache License
+
```
Comment (sdake) [inline LICENSE]: this is the change I was talking about.  I guess the ASL modified their license every so slightly.  big groaner there.

**Pull request merged**